assets/src/stories-editor/blocks/amp-story-cta/index.js

```diff
@@ -36,6 +32,14 @@ const schema = {
 		type: 'string',
 		default: '#32373c',
 	},
+	btnPositionTop: {
+		type: 'number',
+		default: 0,
+	},
+	btnPositionLeft: {
+		type: 'number',
+		default: 0,
```

Comment:
We should probably use better defaults here and deprecate the previous block instead (to avoid validation errors).

(If we keep the default in `0` the update won't cause validation errors, however, the previous CTA blocks would display in the left top corner of the CTA layer which isn't necessarily the best option either.)